app/src/main/java/com/rockthevote/grommet/ui/eventFlow/PartnerLoginViewModel.kt

```diff
@@ -47,35 +47,31 @@ class PartnerLoginViewModel(
         } else {
             viewModelScope.launch(dispatchers.io + coroutineExceptionHandler) {
                 runCatching {
-                    rockyService.getPartnerName(partnerId.toString()).toBlocking().value()
-
-                }.onSuccess {
-                    if (it.response()!!.isSuccessful) {
```

Comment:
This threw an exception causing the app to hang during validation. `getPartnerName` returned a failed result, so `response` was null.

Reply:
👍 good catch